x-pack/plugins/security_solution/cypress/tasks/timeline.ts

```diff
@@ -90,7 +90,9 @@ export const addNameAndDescriptionToTimeline = (timeline: Timeline) => {
 };
 
 export const goToNotesTab = () => {
-  return cy.get(NOTES_TAB_BUTTON).click({ force: true });
+  cy.get(NOTES_TAB_BUTTON)
+    .pipe(($el) => $el.trigger('click'))
```

Comment:
Pipe is the way indeed.

<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/116170416-0d304480-a6c4-11eb-9c66-ca4ee4246dd7.png" />
</p>